packages/neuron-wallet/src/cell.ts

```diff
@@ -6,6 +6,22 @@ export interface OutPoint {
   index: number
 }
 
+interface TypeScript {
+  version: number
+  args: string[]
+  signedArgs: string[]
+  reference: string | null
```

Comment:
reference: string | undefined, and better not to use null [guide](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined)

Reply:
OK

Reply:
should accept null as well, the RPC result will set to null, it should be compatible